conf/waivers/20-long-term

```diff
@@ -5,16 +5,6 @@
 # ie. failures that we don't expect to fix until a big feature is implemented,
 # even if it might take a year or two for the feature to be done
 
-# requires running firewalld (firewall-cmd) and NetworkManager,
-# which are not available in their final form in the Anaconda environment
-# - see https://github.com/ComplianceAsCode/content/issues/9746
-/hardening/anaconda(/with-gui)?/[^/]+/firewalld_sshd_port_enabled
-# - possibly unrelated https://github.com/ComplianceAsCode/content/issues/12276
```

Comment:
I don't see `firewalld_sshd_port_enabled` failing in the stabilization results, should we close https://github.com/ComplianceAsCode/content/issues/12276 ?